docs/website/docs/reference/tuning.md

```diff
@@ -1,36 +1,47 @@
 ---
+hide:
+  - tags
+tags:
+  - sharktuner
+  - SHARKTUNER
+  - Tuner
```

Comment:
Pick 1 tag or none, not 3. Given that there will likely be only 1-2 pages for the tuner, I would vote to instead tag this "performance" or "optimization" then add that tag to other pages that have a similar focus (benchmarking guides, optimization options pages, etc.).

See https://iree.dev/community/tags/ for all current tags.